### PR TITLE
Fixes #30937 - be dynflow client on post migration check

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: lib/katello/tasks/pulp3_post_migration_check.rake [hunk 1]
```diff
@@ -1,8 +1,9 @@
 require File.expand_path("../engine", File.dirname(__FILE__))
+load "#{Katello::Engine.root}/lib/katello/tasks/common.rake"
 
 namespace :katello do
   desc "Runs a post Pulp3 migration check for supported content types."
-  task :pulp3_post_migration_check => :environment do
+  task :pulp3_post_migration_check => [:environment, :disable_dynflow] do
```
Comment (adamruzicka) [inline lib/katello/tasks/pulp3_post_migration_check.rake:6]: Would be cool if Katello could switch over to the definition of this from Foreman https://github.com/theforeman/foreman/blob/develop/lib/tasks/dynflow.rake#L33 and eventually got rid of the katello-specific way of doing things

Comment (jlsherrill) [reply lib/katello/tasks/pulp3_post_migration_check.rake:6]: yes! thank you for mentioning this.  I will open an issue and put it on our sprint to change this across the board.  Will leave this change as is for now

**Review by ianballou — APPROVED** [review]
ACK pending jenkins!

File: lib/katello/tasks/pulp3_post_migration_check.rake (hunk 1 above)
Comment (jlsherrill) [reply lib/katello/tasks/pulp3_post_migration_check.rake:6]: opened https://projects.theforeman.org/issues/30941

**Pull request merged**